### DISCUSSION TITLE
第1节课笔记(23班)

### DISCUSSION THREAD

**Discussion by wangyijunlyy** [InternLM/Tutorial#165] [笔记]
请提交笔记链接

**Comment by relishLeisure** [comment]
http://t.csdnimg.cn/kwNaJ

**Reply by yaoyaotaba** [reply]
https://zhuanlan.zhihu.com/p/676942976

**Reply by yaoyaotaba** [reply]
知乎调整到问答里了，原来的文章链接失效了。新的链接：
https://www.zhihu.com/question/623102115/answer/3356283091

**Comment by educatedyouth** [comment]
https://blog.csdn.net/qq_52603277/article/details/135475879?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135475879%22%2C%22source%22%3A%22qq_52603277%22%7D

**Comment by yayayayaye** [comment]
https://b3x9893p70.feishu.cn/docx/PA8DdbSoEoegeBxiyRacIyTsnXf?from=from_copylink

**Comment by Ace-may** [comment]
https://t5b1jxp2wd.feishu.cn/docx/IDSzdm4mfozRvRxVnIEcgxaDnTg?from=from_copylink

**Comment by i-square** [comment]
https://i-square.github.io/p/InternLM-tutorial-campsection1-LLM-Full-Stack-Open-Source-Ecosystem/

**Comment by YangZhC** [comment]
https://blog.csdn.net/donotpython/article/details/135490437?spm=1001.2014.3001.5502

**Comment by Epi10gue** [comment]
http://t.csdnimg.cn/nHqq6

**Comment by GZdoudou9** [comment]
https://jdpx99pe95.feishu.cn/docx/HKANdoKlIoOsETxqh32ckF31nBd?from=from_copylink

**Comment by PlateAss** [comment]
https://plateass.github.io/class1

**Comment by RX2022-GAN** [comment]
https://space.bilibili.com/v/note-list

**Comment by 1131764933** [comment]
https://shimo.im/file-invite/dxyUePbP4c7ryisvaeS85zX0A84o6/ 颜洪斌 邀请您协作《书生·浦语大模型全链路开源体系》，点击链接开始编辑

**Comment by young1881** [comment]
https://disco-poison-607.notion.site/cad7b5183d1b4c1e9937de345873d4c7?pvs=4

**Comment by Jialu-Yu** [comment]
https://szanrcv530.feishu.cn/docx/KWiLdwAnMo7tYexGBiQccYPfnJ9

**Comment by kingwkun** [comment]
http://t.csdnimg.cn/7kS4A

**Comment by Midkey** [comment]
https://blog.csdn.net/qq_24965035/article/details/135535313?spm=1001.2014.3001.5502
